spec/lib/curation_concerns/null_logger_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe CurationConcerns::NullLogger do
+  subject { described_class.new }
+  context "with an info message" do
```

Comment:
I'm not sure what this `context` block adds and I find it a bit confusing -- I wouldn't call `logger.error` to be an "info" message. But I dig the PR and I wouldn't hold it up over this.